src/librbd/operation/ResizeRequest.h

```diff
@@ -74,11 +74,8 @@ class ResizeRequest : public Request<ImageCtxT> {
    *    |\--------> STATE_GROW_OBJECT_MAP (skip if object map
    *    |                 |                disabled)
    *    |                 v
-   *    |           STATE_POST_BLOCK_WRITES
```

Comment:
@dillaman "(unblock writes)" at the line 70 seems to be not correct now (should be moved to "shrink" branch)?

Reply:
Thx -- tweaked